data/hai missions.txt

```diff
@@ -435,3 +435,566 @@ mission "Unfettered returning home"
+				accept
+			
+	on complete
+		payment 45000
+		conversation
+			`Elliot thanks your for bringing him to <planet> and hands you <payment>. He flags down a taxi just outside the spaceport, but before entering the taxi he turns back to you.`
+			`	"Do you want to come with me? I'd love for you to meet my family. They'll want to meet you when I tell them how I got back."`
+			choice
+				`	"Sorry, I have other places to be. Good luck."`
+				`	"Sure. I've come with you this far."`
+					goto sure
+			
+			`	"I understand. Thanks again, <first>. I'll always owe you."`
+			`	Elliot gets into the taxi, which drives off toward the seemingly endless farmlands outside of the spaceport city.`
+				decline
+			
+			label sure
+			`	The view for nearly the entire three hour drive is of hills and farmland with the occasional building or wild animal. Elliot shakes as the taxi approaches the house from the driveway. "I'm wondering if they still even live here," he says nervously.`
+			`	No one comes out of the house when the taxi stops. You walk up to the front door with Elliot and a middle aged woman answers the door with a mug of coffee and a bewildered look on her face.`
+			`	"Can I help you?" the woman asks.`
+			`	Eliiot looks over at you with a big smile on his face. "Yes you can, mom," he says to her. The woman's face lights up with joy and she begins jumping up and down, spilling coffee from her mug.`
+			`	"Oh Elliot, I can't believe it!" Elliot's mom puts down her mug and leaps out the door to hug him. "Jackson! Sarah! Abby! Sean! Victor! Elliot is here!" Elliot's entire family comes out from the house and greets him, asking dozens of questions about where and how he's been and why he left.`
+			`	Elliot introduces you to his family, who thank you for bringing Elliot home. Victor, who was only seven years old when Elliot left and is now twelve, asks you question after question about being a pilot, sometimes asking new questions before you can even answer the old one.`
+			`	"Are you going back to the spaceport, <first>?" the taxi driver yells from the driveway. "I need to get back to the city as soon as I can."
+			`	You say goodbye to Elliot and to his family. "Thanks again, <first>," Elliot says. "I'll always owe you."`
+
+
+
+mission "Unwanted Cargo Trigger"
+	invisible
+	landing
+	source
+		government "Hai"
+	to offer
+		random < 10
+		"cargo space" > 0
+	on offer
+		event "hai in cargo" 14
+		fail
+
+event "hai in cargo"
+
+mission "Unwanted Cargo"
+	landing
+	description "Bring a Hai child who hid in your cargo hold back to <destination>."
+	source
+		government "Republic" "Free Worlds" "Syndicate" "Neutral"
+	destination "Allhome"
+	to offer
+		has "event: hai in cargo"
+	on offer
+		conversation
+			`The check cargo light for your ship begins blinking when you land, meaning that a cargo crate may have come loose in flight. You shut off the ship's engines and walk to the cargo hold.`
```

Comment:
"check cargo light" reads a little awkwardly, I know what you mean but it still took me several attempts to get right.

Reply:
Yeah, I know what you mean. I wasn't entirely sure about this myself when I wrote it but couldn't think of an alternative. Do you have any suggestions as to what I should change it to.

Reply:
The 'Check Cargo' light, is what I think you wanted to type